src/PHPDocsMD/Console/PHPDocsMDCommand.php

```diff
@@ -145,6 +153,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
                         $docs .= '> '.$class->getDescription().PHP_EOL.PHP_EOL;
                 }
 
+                if ($includeSee && $seeArray = $class->getSee()) {
+                    foreach ($seeArray as $see) {
+                        $docs .= 'See ' . $see . '<br />' . PHP_EOL;
```

Comment:
a simple `<br>` should right, html5 style.

Reply:
That's also a topic for the other files in this PR.

Reply:
I went with "<br />" to match the already present "<hr />" at line 144. But it's all good, I'll change it.

Reply:
On second thought I'll leave it as is until the maintainer requests differently, at which point would be better to change all instances.